unstructured_inference/yolox_functions.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Code based on data_augment.py, demo_utils.py from yolox
```

Comment:
Per the terms of the Apache 2.0 license, we should say more prominently that this code has been modified from the original source 

>       (b) You must cause any modified files to carry prominent notices
          stating that You changed the files; and

Reply:
I solved according: https://softwareengineering.stackexchange.com/questions/220068/file-with-apache-2-0-and-my-modifications 
The new version adds a notice below original license. 
I am not sure, but maybe is also needed to add a file to our repo with the license of YoloX.